twitter_ads/resource.py

```diff
@@ -42,12 +42,29 @@ def __init__(self, account):
     def account(self):
         return self._account
 
-    def from_response(self, response):
+    def from_response(self, response, headers=None):
```

Comment:
Adding a new argument `headers` where caller can pass `response.headers` as an optional argument.

Currently, caller passes only response body:
```python
return self.from_response(response.body['data'])
```
so adding response headers in order to process rate limit headers:
```python
return self.from_response(response.body['data'], response.headers)
```

Reply:
Just reminder to follow up about debugging level headers and exposing those as well (X- headers)

Reply:
Yup, we can generalize the extract_rate_limit() method in the future to handle more.

Reply:
+1 to John's point. I believe there are plans to include  rate limit headers to the async stats create endpoint. Given the current setup, this should be fairly trivial to add. Good stuff!

Reply:
Thanks! I just changed the func name to `extract_response_headers()` so this can be consistent for future use such as async one @tushdante mentioned.